bin/fuel-core/chainspec/local-testnet/chain_config.json

```diff
@@ -45,255 +45,255 @@
           "addi": 2,
           "and": 2,
           "andi": 2,
-          "bal": 274,
+          "bal": 49,
           "bhei": 2,
-          "bhsh": 32,
-          "burn": 7566,
+          "bhsh": 153,
+          "burn": 19164,
           "cb": 2,
           "cfsi": 2,
           "div": 2,
           "divi": 2,
-          "eck1": 1489,
-          "ecr1": 20513,
+          "eck1": 3914,
+          "ecr1": 51339,
           "eq": 2,
           "exp": 2,
           "expi": 2,
           "flag": 1,
           "gm": 2,
           "gt": 2,
-          "gtf": 3,
-          "ji": 2,
-          "jmp": 2,
+          "gtf": 4,
+          "ji": 1,
+          "jmp": 1,
           "jne": 2,
-          "jnei": 2,
-          "jnzi": 2,
-          "jmpf": 2,
-          "jmpb": 2,
-          "jnzf": 2,
-          "jnzb": 2,
-          "jnef": 2,
-          "jneb": 2,
+          "jnei": 1,
+          "jnzi": 1,
+          "jmpf": 1,
+          "jmpb": 1,
+          "jnzf": 1,
+          "jnzb": 1,
+          "jnef": 1,
+          "jneb": 1,
```

Comment:
We don't run benchmarks for all opcodes. Only for several of them:


<img width="1049" alt="image" src="https://github.com/user-attachments/assets/bd21baec-8fff-4460-961a-44046c5c4a57" />

In the past, I was updating manually them to much one value. Maybe we should add benchmarks for them, or update `collect` binary to use the same value for them=)

Reply:
@FuelLabs/client Who wants to do that?=D

Reply:
I can do that tomorrow morning. If I understand well some of the opcode such as `jnzf` should have a benchmark but don't have and we want them to have one now or use the same value as another opcode that is doing something similar to the missing one ?